clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -285,9 +284,8 @@ private void seekBlockInStream(long newPos) throws IOException {
       // Reading next block entirely.
       if (mPos % mBlockSize == 0 && mShouldCacheCurrentBlock) {
         try {
-          mCurrentCacheStream =
-              mContext.getTachyonBlockStore().getOutStream(currentBlockId, -1,
-                      NetworkAddressUtils.getLocalHostName(ClientContext.getConf()));
+          mCurrentCacheStream = mContext.getTachyonBlockStore().getOutStream(currentBlockId, -1,
+              NetworkAddressUtils.getLocalHostName(ClientContext.getConf()));
```

Comment:
Hm, I re-downloaded http://tachyon-project.org/documentation/resources/tachyon-code-formatter-eclipse.xml and re-imported it, and it still wants to make this change. Why use an extra line when it doesn't improve the position of the line break?